NDB.Covid19/NDB.Covid19.Droid/Resources/layout/farewell_smittestop_page.xml

```diff
@@ -21,12 +21,15 @@
         android:id="@+id/scroll_view"
         style="@style/ScrollScreen"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
+        android:layout_height="0dp"
+	    android:layout_weight="1"
```

Comment:
What happened to the indentation here? 🧐

Reply:
Looked the same in the editor. Found out one used some spaces, the other used only tabs. Which github shows different ^^ 
Fixed now!